frontend/app/src/components/home/PollBuilder.svelte

```diff
@@ -88,7 +91,7 @@
 
     function deleteAnswer(answer: string) {
         poll.pollAnswers.delete(answer);
```

Comment:
This should now be `poll.pollAnswers.delete(answer.toUpperCase())` shouldn't it?